### PR TITLE
[MIG][11.0] project_task_material

### PR DESCRIPTION
**Pull request by ghost**
Tasks done:

- Deleted "utf-8" unnecessary first-lines.
- Added new tests for simulating user and manager task material in-line creation.

### CODE REVIEW

**Comment by ghost** [comment]
As spanish people said "_A la tercera va la vencida_". Third time lucky.

File: project_task_material/models/project.py
Comment (pedrobaeza) [inline project_task_material/models/project.py]: Don't change this case

File: project_task_material/security/ir.model.access.csv
Comment (pedrobaeza) [inline project_task_material/security/ir.model.access.csv]: Don't remove extra line

File: project_task_material/views/project_view.xml
Comment (pedrobaeza) [inline project_task_material/views/project_view.xml]: Don't change this

File: project_task_material/views/project_view.xml
Comment (pedrobaeza) [inline project_task_material/views/project_view.xml]: Don't remove final line

File: project_task_material/tests/test_create_material_lines.py
Comment (pedrobaeza) [inline project_task_material/tests/test_create_material_lines.py]: Save a variable with `task = self.task.sudo(self.project_user.id)` to avoid the same call too much times.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Several comments apply in all the code.

**Comment by ghost** [comment]
@pedrobaeza I have already made that changes, could you review it? Thank you very much.

File: project_task_material/README.rst
Comment (PCatinean) [inline project_task_material/README.rst]: Update to the new README.rst format from (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst):
```
.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
   :target: https://www.gnu.org/licenses/agpl
   :alt: License: AGPL-3
```

File: project_task_material/README.rst
Comment (PCatinean) [inline project_task_material/README.rst:39]:
```
Images
------

* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
```

File: project_task_material/README.rst
Comment (PCatinean) [inline project_task_material/README.rst:53]:
```
Do not contact contributors directly about support or help with technical issues.
```

**Comment by PCatinean** [comment]
Tested on runbot, LGTM. Approve once small changed to README are done.

**Review by PCatinean — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@PCatinean @pedrobaeza I did the changes you requested me, I think I forgot nothing.
Thank you all.

**Review by PCatinean — APPROVED** [review]

**Comment by ghost** [comment]
@pedrobaeza @chienandalu Could you take a new review please? Thank you very much.

**Review by rodrigets — APPROVED** [review]

File: project_task_material/__manifest__.py
Comment (pedrobaeza) [inline project_task_material/__manifest__.py]: Why do you remove commas? Don't touch what is not needed. And your style is worse then the previous one.

File: project_task_material/__manifest__.py
Comment (pedrobaeza) [inline project_task_material/__manifest__.py]: This migration doesn't represent too much work, so you should add as contributor, but not being promoted to co-author.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I see also a lot of test overhead that is not needed. You don't need to test permissions. That's something of the ORM. You have to test the module code.

**Comment by ghost** [comment]
Thank you @pedrobaeza , you have a lot of patience with me, I am a beginner in this world and I would like to learn about every gear in odoo. I appreciate it.

**Review by dreispt — APPROVED** [review]
LGTM

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]
Thanks for your efforts!

**Pull request merged**
